CHANGELOG.md

```diff
@@ -55,6 +55,9 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * ``TreeNode.to_array`` now supports replacing ``nan`` branch lengths in the resulting branch length vector with the value provided as ``nan_length_value``.
 * ``skbio.io.format.phylip`` now supports sniffing and reading strict, sequential PHYLIP-formatted files into ``skbio.Alignment`` objects. ([#1006](https://github.com/biocore/scikit-bio/issues/1006))
 * Added `default_gap_char` class property to ``DNA``, ``RNA``, and ``Protein`` for representing gap characters in a new sequence.
+* Added ``skbio.sequence.GrammaredSequence``, which can be inherited from to create grammared sequences with custom alphabets (e.g., for use with TabularMSA) (#1175)
```

Comment:
These changelog entries are in the wrong location (0.4.1). Can you please move them under 0.4.1-dev? I think the first two bullet points should go under "Features" and the third bullet point (DNA/RNA/Protein subclassing) should go under "Bug fixes".